addon/mixins/select.js

```diff
@@ -44,7 +42,7 @@ export default Mixin.create({
 
     let path = this.get('_labelPath');
     if (path && isPresent(checkValue)) {
-      return (typeof checkValue.get === 'function' ? checkValue.get(path) : void 0) || checkValue[path]
+      return checkValue.get ? checkValue.get(path) : checkValue[path]
```

Comment:
does `checkValue.get` work as the `typeof checkValue.get === 'function' ` check?

Reply:
Nah, it just checks whether `checkValue.get` exists at all, but my theory was that if it exists, and it's not a function, someone did something wrong. But I actually backed out of this and did a force push, so this is outdated.